src/lib.rs

```diff
@@ -242,6 +251,19 @@ where
             .map_err(|_| Error::DisplayError)
     }
 
+    fn framebuffer_size(&self) -> (u16, u16)
```

Comment:
Thought about making this return a `Size` object. That would clarify which one is width vs height. But Size objects use u32 dimensions. And `pixel_count` in `graphics.rs` uses `usize` dimensions. And `usize::from` works fine for u16, but fails for u32. So I stuck with the above for now.

Reply:
`u32 as usize` should be safe in 32bit+ machines of course there are very special edge cases (there's a whole thing with ptrsize vs usize in rust). for our purposes and expected display sizes I think it's 100% safe to just case that way.